modules/persistent-storage-csi-secrets-store-driver-install.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-secrets-store-driver-install_{context}"]
+= Installing the Secrets Store CSI Driver
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Administrator access to the cluster.
+
+.Procedure
+
+To install the Secrets Store CSI Driver:
+
+. Install the Secrets Store Container Storage Interface (CSI) driver operator:
+.. Log in to the web console.
+.. Click *Operators* → *OperatorHub*.
+.. Locate the Secrets Store CSI Operator by typing Secrets Store CSI in the filter box.
+.. Click the *Secrets Store CSI Driver Operator* button.
+.. On the *Secrets Store CSI Driver Operator* page, click *Install*.
+.. On the *Install Operator* page, ensure that:
++
+* *All namespaces on the cluster (default)* is selected.
+
+* *Installed Namespace* is set to *openshift-cluster-csi-drivers*.
+.. Click *Install*.
++
+After the installation finishes, the Secrets Store CSI Operator is listed in the *Installed Operators* section of the web console.
+
+. Create the `ClusterCSIDriver` instance for the driver (`secrets-store.csi.k8s.io`):
+.. Click *Administration* -> *CustomResourceDefinitions* -> *ClusterCSIDriver*.
+.. On the *Instances* tab, click *Create ClusterCSIDriver*.
++
+Use the following YAML file:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: ClusterCSIDriver
+metadata:
+    name: secrets-store.csi.k8s.io
+spec:
+  logLevel: Normal
+  managementState: Managed
+  operatorLogLevel: Trace
+----
+
+.. Click *Create*.
+
+. Install a third-party provider plugin for your chosen secret store.??? link to authentication content????
```

Comment:
This is where you could create a commented out TODO, and I can come in to help fill this next step piece in with my PR